iarm_query_powerstate/IARM_Bus_CheckPowerStatus.c

```diff
@@ -23,14 +23,23 @@
 #include <string.h>
 #include <unistd.h>
 
-// This include is still required for data types to read cached power status residing in /opt/uimgr_settings.bin
-// TODO: after IARM PwrMgr logic is moved to PowerManager plugin refactor this code and remove this header file too.
-#include "pwrMgr.h"
-
 #include "power_controller.h"
 
 #define PADDING_SIZE 32
 
+/**
+ * @brief All possible power states
+*/
+typedef enum _Daemon_PowerState_t
```

Comment:
Why we need to define here? should it derived from the original place from power controller?